scripts/find.rb

```diff
@@ -0,0 +1 @@
+Person.where(age: > ?, age > 20)
```

Comment:
Close, but with the string to symbol conversion method the syntax is: `Person.where('age > :age',{age: 20})
`